### PR TITLE
Fix doc for CTC_Greedy

### PR DESCRIPTION
**Pull request by Garvys**
The output is the minus the sum of the logits and the logprobs

### CODE REVIEW

**Comment by Garvys** [comment]
I signed it!

File: tensorflow/python/ops/ctc_ops.py [hunk 1]
```diff
@@ -206,8 +206,8 @@ def ctc_greedy_decoder(inputs, sequence_length, merge_repeated=True):
         The vector stores the decoded classes.
       `decoded.shape`: Shape vector, size `(2)`.
         The shape values are: `[batch_size, max_decoded_length]`
-    log_probability: A `float` matrix `(batch_size x 1)` containing sequence
-        log-probabilities.
+    neg_sum_logits: A `float` matrix `(batch_size x 1)` containing, for the
+        sequence found, minus the sum of the greatest logits.
```
Comment (martinwicke) [inline tensorflow/python/ops/ctc_ops.py]: This sentence seems to be missing something

**Review by martinwicke — CHANGES_REQUESTED** [review]

**Comment by martinwicke** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
@Garvys feel free to address the comments.

**Review by martinwicke — APPROVED** [review]

**Comment by martinwicke** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
Ignoring flaky `queue_runner_test`.
